var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -22,11 +22,12 @@ class Libxsmm(MakefilePackage):
 
     license("BSD-3-Clause")
 
+    version("develop", branch="main")
+
```

Comment:
I think it's worth closing. As you pointed out we have a syntax for literal versions `@=main` vs `@=main-2023-11`